projects/storefrontlib/src/cms-components/checkout/services/checkout-config.service.ts

```diff
@@ -1,14 +1,20 @@
 import { Injectable } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { RoutingConfigService } from '@spartacus/core';
-import { CheckoutConfig } from '../config/checkout-config';
+import { DeliveryMode, RoutingConfigService } from '@spartacus/core';
+import {
+  CheckoutConfig,
+  DeliveryModePreferences,
+} from '../config/checkout-config';
 import { CheckoutStep, CheckoutStepType } from '../model/checkout-step.model';
 
 @Injectable({
   providedIn: 'root',
 })
 export class CheckoutConfigService {
   steps: CheckoutStep[] = this.checkoutConfig.checkout.steps;
+  express: boolean = this.checkoutConfig.checkout.express;
```

Comment:
do we need this and the next one as a public API?
Also those values are not marked as readonly, so is it ok to change them at runtime by customer?